open-banking-brasil-certificate-standards-1_ID1.md

```diff
@@ -331,7 +334,7 @@ otherName.3 = 2.16.76.1.3.7;UTF8:<INSS Number>
 | NA | OIDC | pushed_authorization_request_endpoint |  ICP WEB SSL | Required |
 | NA | DCR | registration_endpoint |  ICP WEB SSL | Required |
 | NA | OIDC | revocation_endpoint | ICP WEB SSL | Required |
-| NA | OIDC | introspection_endpoint _(*)_ | ICP WEB SSL | Required |
+| NA | OIDC | introspection_endpoint _(**)_ | ICP WEB SSL | Required |
```

Comment:
you really don't want to be making your introspection endpoints available to TPPs if you don't have too. So for security i would remove this completely.

Reply:
I don´t  :-) 

_(**)_ this endpoint shall not be published to TTPs, but if publishing this endpoint is required to deal with an special use case (**which is not mandatory nor recommended for security reasons**), it shall be made using ICP WEB SSL and  require MTLS authentication.

Reply:
So i know that all banks will fail this internally ;-) the majority of API Gateways won't support mutual tls for introspeciting tokens against the token endpoint. But! Since we can't write a test for it you could leave it in.